native/src/routes/Pois.tsx

```diff
@@ -87,43 +94,74 @@ const Pois = ({ pois, language, cityModel, route, navigation }: PoisProps): Reac
       }
     }, RESTORE_TIMEOUT)
   }
-  const selectPoiFeature = (feature: PoiFeature | null) => {
-    if (feature && cameraRef.current) {
-      setFollowUserLocation(false)
-      navigation.setParams({ slug: feature.properties.slug })
+
+  const deselectFeature = useCallback(() => {
+    if (multipoi && currentPoi) {
+      navigation.setParams({ slug: undefined })
+    } else {
+      navigation.setParams({ slug: undefined, multipoi: undefined })
+      scrollTo(listScrollPosition)
+    }
+  }, [currentPoi, listScrollPosition, multipoi, navigation])
+  useOnBackNavigation(slug || multipoi ? deselectFeature : undefined)
+
+  const selectFeatureOnMap = (feature: PoiFeature | null) => {
+    if (!cameraRef.current) {
+      return
+    }
```

Comment:
Theoretically because, the camera needs to move to the Poi location. If the ref is undefined nothing would happen. I removed this, as we can probably just not care if the ref exists. This should be only relevant until the 'dom' is fully loaded. So only bots, should be able to tap that fast.